plugin/declarativenetworkmanager.h

```diff
@@ -157,6 +169,10 @@ public Q_SLOTS:
     void serviceCreated(const QString &servicePath);
     void serviceCreationFailed(const QString &error);
 
+    void tetheringClientsChanged();
+    void tetheringClientAdded(const QString &macAddress);
+    void tetheringClientRemoved(const QString &macAddress);
```

Comment:
If not sure about these in c++, here I'd outright just remove. The property change signal should be enough.